org.sonarlint.eclipse.ui/src/org/sonarlint/eclipse/ui/internal/markers/DeactivateRuleMarkerResolver.java

```diff
@@ -41,7 +41,8 @@ public String getDescription() {
 
   @Override
   public String getLabel() {
-    return "Deactivate rule " + marker.getAttribute(MarkerUtils.SONAR_MARKER_RULE_KEY_ATTR, "unknown");
+    // U+200B is zero-width space; a hack to make the display order of this resolver come after than other items
+    return "\u200bDeactivate rule " + marker.getAttribute(MarkerUtils.SONAR_MARKER_RULE_KEY_ATTR, "unknown");
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this Unicode escape sequence and use the character instead. ([squid:S2973](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS2973))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.sonarlint.eclipse%3Asonarlint-eclipse-parent&pullRequest=153&issues=AWLzSLcNTcYTQXsfsDi0&open=AWLzSLcNTcYTQXsfsDi0)

[ISSUE_KEY]: AWLzSLcNTcYTQXsfsDi0